code/game/machinery/doors/airlock.dm

```diff
@@ -611,11 +611,9 @@ var/list/airlock_overlays = list()
 			if(!istype(H.head, /obj/item/clothing/head/helmet))
 				H.visible_message("<span class='danger'>[user] headbutts the airlock.</span>", \
 									"<span class='userdanger'>You headbutt the airlock!</span>")
-				var/obj/item/bodypart/affecting = H.get_bodypart("head")
 				H.Stun(5)
 				H.Weaken(5)
-				if(affecting && affecting.take_damage(10, 0))
-					H.update_damage_overlays(0)
+				H.apply_damage(10, BRUTE, "head")
```

Comment:
With this, the damage is transfered to the chest if the head is missing (e.g. retarded headless changeling bumping door)